lib/tasks/translation.rake

```diff
@@ -83,4 +83,19 @@ namespace :translation do
     end
   end
 
+  desc "Add missing translations"
+  task(:add_missing, [:locale] => [:environment]) do |t, args|
+    locale = args[:locale] || "en"
+    require 'i18n/tasks/cli'
+
+    I18n::Tasks::CLI.start(["add-missing", "-l", locale])
```

Comment:
We probably want to make the `-l` flag optional, incase we want to add keys to multiple locales. There is also another flag `--nil-value` we will want to use, otherwise I18n-tasks tries to fill in values for all the keys it adds. Sorry, I should've mentioned the exact flags we intent to use in the Trello card.